src/component/common/Splash/Splash.styles.ts

```diff
@@ -0,0 +1,66 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles(theme => ({
+    mainContainer: {
+        backgroundColor: theme.palette.primary.light,
+        height: '100vh',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    container: {
+        backgroundColor: theme.palette.primary.main,
+        position: 'relative',
+        height: '85vh',
```

Comment:
I think we are better off using more defined values, as this could quickly look weird on 4k screens.